[vtk] Add export header for VTK::doubleconversion

This is usually generated by CMake, but it's easier to just write it
by hand for our Bazel build.

TODO(jwnimmer-tri) Enhance settings.bzl to be able to express this
directly.

--- /dev/null
+++ ThirdParty/doubleconversion/vtkdoubleconversion/double-conversion/vtkdoubleconversion_export.h
@@ -0,0 +1,2 @@
+#pragma once
+#define VTKDOUBLECONVERSION_EXPORT
